src/ontology/config/prefixes.csv

```diff
@@ -224,8 +224,19 @@ PLANP,http://purl.obolibrary.org/obo/PLANP_
 CARO,http://purl.obolibrary.org/obo/CARO_
 NPO,http://purl.bioontology.org/ontology/npo#NPO_
 ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
+ICD10CM2,https://icd.codes/icd10cm/
```

Comment:
## FYI: New entries to `prefixes.csv`
<details><summary>Details</summary>
<p>

Addresses [this issue](https://github.com/monarch-initiative/mondo-ingest/pull/363#discussion_r1327876041).

@matentzn You may want to look over these prefixes.
Some were missing, and I simply added.
But some are duplicates of other prefixes with different URI stems:

```
ICD10WHO,https://icd.who.int/browse10/2019/en#/
ICD10WHO2010,http://apps.who.int/classifications/icd10/browse/2010/en#/
MEDGEN,http://purl.obolibrary.org/obo/MEDGEN_
MEDGEN2,http://identifiers.org/medgen/
UMLS,http://purl.obolibrary.org/obo/UMLS_
UMLS2,http://linkedlifedata.com/resource/umls/id/
OMIMPS,https://www.omim.org/phenotypicSeries/PS
OMIMPS2,https://omim.org/phenotypicSeries/
```

The OMIMPS issue is due to:
- https://github.com/monarch-initiative/omim/issues/103

</p>
</details>

Reply:
Please create a new issue to move all prefix management to curies package, assign to yourselves and then resolve this comment. I don't mind this as a quick fix, but we should really fix our upstream tickets and not have 3 different places for Mondo prefixes (config/mondo.sssom.yml, config/prefixes.csv, curies obo context).

Reply:
Created new issue:
- #146

Reply:
Created new issue:
- #146